libpromises/cf3lex.l

```diff
@@ -116,6 +116,7 @@ arrow      ->
 qstring        \"((\\(.|\n))|[^"\\])*\"|\'((\\(.|\n))|[^'\\])*\'|`[^`]*`
 
 class          [.|&!()a-zA-Z0-9_\200-\377:]+::
+varclass       (\"[^"]*\"|\'[^"]*\')::
```

Comment:
The second half of this regex appears to be broken:surely it should be `\'[^']\'` excluding single quote, instead of double, between the single quote delimiters.